lib/BackbeatConsumer.js

```diff
@@ -110,10 +131,15 @@ class BackbeatConsumer extends EventEmitter {
     _init() {
         if (this._bootstrap) {
             this._consumerReady = true;
-            process.nextTick(this._checkIfReady.bind(this));
         } else {
             this._initConsumer();
         }
+        if (this._kafkaBacklogMetricsConfig) {
+            this._initKafkaBacklogMetrics();
+        } else {
+            this._kafkaBacklogMetricsReady = true;
+        }
+        process.nextTick(this._checkIfReady.bind(this));
```

Comment:
minor: that should only be needed if `this._kafkaBacklogMetricsConfig` is false. (it might come straight from the Zenko code, feel free to ignore)